lms/urls.py

```diff
@@ -103,6 +103,9 @@
     url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', app_name="credit", namespace='credit')),
     url(r'^rss_proxy/', include('rss_proxy.urls', namespace='rss_proxy')),
     url(r'^api/organizations/', include('organizations.urls', namespace='organizations')),
+
+    # Update session view
+    url(r'^update_language_session$', 'lang_pref.views.update_language_session'),
```

Comment:
Is there a prefix we should consider for this endpoint?  "api/", "lang-pref/", etc.?  It seems odd for this to be a top level resource.   Also it would be nicer if we could model this as a resource-oriented endpoint, such as "session_language".